### PR TITLE
Bergado

### PR DESCRIPTION
**Pull request by bergado-shamma**

### CODE REVIEW

File: Bergado_Shamma/script.js
Comment (Sanchez-SamanthaNicole) [inline Bergado_Shamma/script.js:1]: Some lines exceed 80 columns. Check your code again.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0

File: Bergado_Shamma/script.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+document.addEventListener("DOMContentLoaded", function () {
+  let count = 0;
+  let timerInterval;
+  let timerDisplay = document.getElementById("timer");
+  let countList = document.getElementById("count_list");
+
+  function startTimer() {
```
Comment (Sanchez-SamanthaNicole) [inline Bergado_Shamma/script.js:7]: I suggest your startTimer function to make it a guard clause approach. 

function startTimer() {
  if (timerInterval) return; // Guard clause
  
  timerInterval = setInterval(() => {
    count++;
    timerDisplay.textContent = count;
  }, 1000);
}

**Review by Sanchez-SamanthaNicole — CHANGES_REQUESTED** [review]

File: Bergado_Shamma/script.js
Comment (bergado-shamma) [reply Bergado_Shamma/script.js:1]: Thank you po!

File: Bergado_Shamma/script.js (hunk 1 above)
Comment (bergado-shamma) [reply Bergado_Shamma/script.js:7]: Thank you po!

**Comment by sammynics** [comment]
You can now merge your changes and upload it to your page in Netlify. Also, update your README file with your Netlify URL. Thank you!

**Review by Sanchez-SamanthaNicole — APPROVED** [review]
You can now merge your changes and upload it to your page in Netlify. Also, update your README file with your Netlify URL. Thank you!

**Pull request merged**